### ISSUE TITLE
Could you make it work with react 17.0?

### ISSUE THREAD

**Issue by baochungit** [mientjan/react-native-markdown-renderer#166]
I'm using react native with react version 17.0.1 and it seems not working well together.
Could you spend a little time to make it work?
Thanks a lot!

**Comment by julestruong** [comment]
this repo seems pretty dead

**Comment by baochungit** [comment]
Yes, it's dead. Use my fork instead. Add `"react-native-markdown-renderer": "github:baochungit/react-native-markdown-renderer#master"` to your `package.json`
